app/build.gradle

```diff
@@ -49,6 +49,15 @@ dependencies {
     implementation 'com.google.android.gms:play-services-maps:11.8.0'
     implementation 'com.google.android.gms:play-services-location:11.8.0'
 
+<<<<<<< HEAD
```

Comment:
These lines are causing your gradle build to fail. They are here because you merged code. Remove them and push so we can see if the Travis file passes.